serverside/zerotrace/example/main.go

```diff
@@ -115,14 +166,17 @@ func webSocketHandler(w http.ResponseWriter, r *http.Request) {
 		cfg := zerotrace.NewDefaultConfig()
 		cfg.Interface = iface
 		z := zerotrace.NewZeroTrace(cfg)
-		var nwLayerRtt time.Duration
-		nwLayerRtt, err = z.CalcRTT(wssConn)
+		nwLayerRttVal, err := z.CalcRTT(wssConn)
 		if err != nil {
 			l.Println("Error determining RTT with zerotrace", err)
+		} else {
+			nwLayerRtt0T = fmtTimeMs(nwLayerRttVal)
 		}
-		l.Printf("0trace network-layer RTT: %s", nwLayerRtt)
+		l.Printf("0trace network-layer RTT: %s", nwLayerRttVal)
 
-		rttDiff := fmtTimeMs(appLayerRtt.MinRTT) - fmtTimeMs(nwLayerRtt)
+		nwLayerRtt := []float64{nwLayerRttTCP, nwLayerRttICMP, nwLayerRtt0T}
+		l.Println("Min NWLAYERRTT: ", getMinRttValue(nwLayerRtt))
+		rttDiff := appLayerRtt.MinRTT - getMinRttValue(nwLayerRtt)
 		rttDiff = math.Abs(rttDiff)
```

Comment:
Shall we remove this line? I think it's important to preserve how often we end up with a negative RTT difference.

Reply:
Added in [5c7d0c2](https://github.com/reethikar/zerotrace-eval/pull/4/commits/5c7d0c25ed499e2f295de72eb5fab4c251278cd3)